start/cli/src/main/java/org/ethereum/beacon/ReusableOptions.java

```diff
@@ -22,17 +22,17 @@
 public class ReusableOptions {
 
   static final String VERSION = "0.1b";
-  private static final Map<Integer, Level> LOG_LEVELS = new HashMap<>();
+  private static final Map<LogLevel, Level> LOG_LEVELS = new HashMap<>();
 
   static {
-    LOG_LEVELS.put(0, Level.OFF);
-    LOG_LEVELS.put(1, Level.FATAL);
-    LOG_LEVELS.put(2, Level.ERROR);
-    LOG_LEVELS.put(3, Level.WARN);
-    LOG_LEVELS.put(4, Level.INFO);
-    LOG_LEVELS.put(5, Level.DEBUG);
-    LOG_LEVELS.put(6, Level.TRACE);
-    LOG_LEVELS.put(7, Level.ALL);
+    LOG_LEVELS.put(LogLevel.OFF, Level.OFF);
```

Comment:
Do we really need all of them? I am not sure if our code even uses them all. What if we keep those which could be really useful?